platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -576,11 +577,13 @@ check_storage_space() {
         return
     fi
 
-    if ! echo "$application_size" | grep -q '^[0-9]*.[0-9]*$'; then
+    if ! echo "$application_size" | grep -q '^[.0-9]*$'; then
         wrap_bad "Invalid Application Size provided" "$application_size"
         return 2
     fi
 
+    #Provide a Buffer of 50MB to account for any additional log files and storage.
```

Comment:
Nit: This comment shouldn't refer to the `$buffer` variable's underlying value (50MB)

Reply:
Fixed